### ISSUE TITLE
::

### ISSUE THREAD

**Issue by wiztigers** [TypeCobolTeam/TypeCobol#201]
As defined in the [wiki](https://github.com/TypeCobolTeam/TypeCobol/wiki/TypeCobolNameQualification)

**BLOCKED BY #220 #213 #251**

**Comment by smedilol** [comment]
Current situation is acceptable for version 1.0
Limitations are:
- You need to put space before and after the operator (issue #251)
- Operator `::` doesn't handle subscript (need to create a new issue)

**Comment by smedilol** [comment]
All child issues are now created. I close this one.
